release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -290,6 +290,16 @@ In the table, features are marked with the following statuses:
 
 Support for deploying custom schedulers manually has been removed with this release. Use the xref:../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[{secondary-scheduler-operator-full}] instead to deploy a custom secondary scheduler in {product-title}.
 
+[id="ocp-4-11-future-removals"]
+=== Future Kubernetes API removals
+
+The next minor release of {product-title} is expected to use Kubernetes 1.25. Currently, Kubernetes 1.25 plans to remove several deprecated `v1beta1` and `v2beta1` APIs.
+
+See the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Deprecated API Migration Guide] in the Kubernetes documentation for the list of planned Kubernetes API removals.
+
+// TODO: Add a call to action for users now to stop using these APIs?
+// TODO: Do we want to point to the KCS article now? Can admins actually provide the ack now, or will they have to do that right before they actually upgrade?
```

Comment:
@sdodson To keep things simple, can we just add a general call to action for users to start migrating away from these APIs as of 4.11 GA? Like that they will no longer be able to use these APIs in the next release, so they should start migrating away now?

Or do we need to actually link to the KCS article with the actual steps on how to check using the alerts/APIRequestCount, and providing the way to add the admin-ack? (I wasn't sure if they could actually provide the ack early, like at 4.11.0 GA, or if would have to be done right before they upgrade to 4.12).

Reply:
That's a good point, today a call to action, review this list of APIs begin migrating away, we can leave in the details about alerts/APIRequestCount, those will be valid and will help them chase down anything that needs to be cleaned up. However we will leave out specific mention of the ACK for now.

Once we land the admin ack gate in clusters we will amend the KCS with the ack specific bits and clusters will start linking people to the KCS.